site/docs/connectors/resource/jdbc-resource-connector.md

```diff
@@ -0,0 +1,15 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+  
+--8<-- "snippets/content-status/stable.md"
```

Comment:
This is not a stable component - should be marked as in development or technical preview if ready for use

Reply:
Apologies, previously modified another line. Should be the proper one now. Thank you

Reply:
this is fixed